### ISSUE TITLE
Script loading error

### ISSUE THREAD

**Issue by anyway** [ausi/respimagelint#83]
I was really happy to find this tool, but unfortunately in both the current Chrome and Firefox (both on Mac) I get this JS error message in Chrome and nothing but a blank page:

> Refused to load the script 'https://ausi.github.io/respimagelint/collector.js?1674374901488' because it violates the following Content Security Policy directive: "script-src 'self' cybersec.b-cdn.net blob:". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.

Would there perhaps be a workaround for this?

**Comment by ausi** [comment]
See https://github.com/ausi/respimagelint/issues/75#issuecomment-1374481385

**Comment by anyway** [comment]
Okay, thank you... I had overlooked this entry. 

Unfortunately I can't get this fixed.
Neither through the browser settings, nor through header settings on the website. I have also tested this across the board on a few random websites on the web... also without success. That would suggest that it is the browser. But unfortunately it doesn't work in any of my installed browsers: Firefox, Chrome, Safari, Opera, Brave...
